js/extension/features/editing-notepad.js

```diff
@@ -0,0 +1,384 @@
+/**
+ *
+ * Discogs Enhancer
+ *
+ * @author: Ross Lancaster
+ * @github: https://github.com/ToastyMallows
+ *
+ * Puts a notepad in the editing window to store any editing
+ * information like artists names or links relevant to editing on Discogs.
+ */
+rl.ready( () => {
+
+  if ( rl.pageIsNot( 'edit', 'update', 'addRelease', 'drafts' ) ) {
+    // Only add Editing Notepad on editing related pages
+    return;
+  }
+
+  // HTML variables
+  const PREFIX = 'de-editing-notepad';
+  const notepadId = PREFIX;
+  const notepadAreaId = `${PREFIX}-area`;
+  const toggleButtonId = `${PREFIX}-toggle-button`;
+  const editButtonId = `${PREFIX}-edit-button`;
+  const areaWrapperId = `${PREFIX}-area-wrapper`;
+  const notepadEditCoverId = `${PREFIX}-edit-cover`;
+  const notepadEditingClass = `${PREFIX}-isediting`;
+  const notepadFocusedClass = `${PREFIX}-isfocused`;
+  const buttonClass = `${PREFIX}-button`;
+  const buttonsWrapperClass = `${PREFIX}-buttons-wrapper`;
+  const hiddenClass = `${PREFIX}-hidden`;
+  const rotateClass = `${PREFIX}-rotate`;
+  const visibleClass = `${PREFIX}-visible`;
+  const notepadLinkClass = `${PREFIX}-link`;
+  const discogsLoadingClass = "loading-placeholder";
+  const showTitle = 'Show Notepad';
+  const hideTitle = 'Hide Notepad';
+  const editEmoji = '✎';
+  const saveEmoji = '✔';
+  const editTitle = 'Edit';
+  const saveTitle = 'Save';
+  const supportedURISchemes = ['https://', 'http://', 'sftp://', 'file://', 'chrome://', 'smb://']
```

Comment:
I actually just scrolled through [this list](https://en.wikipedia.org/wiki/List_of_URI_schemes) and picked out the ones that I figured were the most common.  This could be expanded to any scheme with regex if we really wanted.  Some don't use `://`, they only use `:`.  I didn't want to accidentally make clickable links when the text being pasted isn't in fact a URI, that's why I limited it to only these.

Reply:
Cool, makes sense. I was wondering if you had a release editing workflow where you might use `sftp` and what not. I don't have any issue with them - just curious :)